cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -105,6 +105,47 @@ public static void writeModelDescription(XMLStreamWriter writer, ModelDescriptio
         writer.writeEndElement();
     }
 
+    private static String toRdfId(String id) {
+        // Handling ids: if received id is not prefixed by "_", add it to make it a valid RDF:Id
+        // We have to be careful with "resource" and "about" references, and apply the same conversions
+        return id.startsWith("_") ? id : "_" + id;
+    }
```

Comment:
Not sure we need that? Either we got the ID from `getUniqueId` which already starts with "_" or we have a IIDM ID and this is to be dealt later to have a RDF-compliant ID.

Reply:
I think we should deal with it. I've introduced the explicit distinction between `mRID` and `rdf:ID` because in CGMES 3 `mRID` is a required attribute now, and it has to be explicit in `EQ.xml` files. 
`CgmesExportUtil.getUniqueId()` should not build ids with a "`_`" as prefix, it should return directly an UUID. 
A more detailed explanation is in #2064.